Jenkinsfile

```diff
@@ -266,15 +266,15 @@ def define_grid_tests() {
   return grid_tests
 }
 
-def define_regression_tests() {
-  regression_tests = [:]
-  regression_tests.failFast = false
+def define_browserstack_tests() {
+  browserstack_tests = [:]
+  browserstack_tests.failFast = false
 
   configurationTypes.each {
     def (config, browser) = it
     def stepName = "${browser} on ${config}"
-    isMobile = (browser == "ios" || browser == "android")
-    regression_tests[stepName] = {
+    def isMobile = (browser == "ios" || browser == "android")
```

Comment:
This was the crux of the issue. For some reason having a global variable here, didn't percolate through and down the correct value of isMobile (Possibly due to default values being set). But using a local variable here works fine?? :shrug: